### ISSUE TITLE
Carthage support

### ISSUE THREAD

**Issue by marbetschar** [gali8/Tesseract-OCR-iOS#297]
Is there Carthage support for this library available?

**Comment by jsl303** [comment]
Second this! I'd love to see Carthage support!!! Please Please!

**Comment by joel-yc** [comment]
Yes please.  We can't use pods in our project, so carthage is a must for us.
